nyc_translate_setup/serge/data/configs/testconfig.serge

```diff
@@ -0,0 +1,502 @@
+        # and converted to single line with no double-spaces?
+        # YES - the string should be normalized
+        # NO - the string is extracted as is (default)
+        normalize_strings           NO
+
+        # (BOOLEAN) [OPTIONAL] Should the string keys reported by a parser
+        # be copied into a context field?
+        # Note that original context in this case becomes overwritten,
+        # unless the returned key is empty.
+        # Changing this setting on an existing previously translated project
+        # will change all the string signatures and may require one to re-apply
+        # and review translations once.
+        # YES - extracted keys should be used as context
+        # NO - keep the original extracted context (default)
+        use_keys_as_context         NO
+
+        # (BOOLEAN) [OPTIONAL] Should untranslated strings appear blank
+        # in the output file, or have source strings as their "translation"?
+        # Rendering source string when a translation is missing may simplify
+        # the internationalization code and optimize your application performance,
+        # while using blank strings can reduce the size of the resource files
+        # (but you will likely need to implement your own fallback logic).
+        # YES - leave the string blank
+        # NO - fall back to source string (default)
+        leave_untranslated_blank    NO
+
+        # (STRING) DBI-compatible connection string
+        # Supported drivers: SQLite, MySQL and Postgres
+        # SQLite sample: 'DBI:SQLite:dbname=/path/to/translate.db3'
+        # SQLite sample: 'DBI:SQLite:dbname=:memory:' - in-memory database
+        # for testing purposes
+        # MySQL sample: 'DBI:mysql:database=translations'
+        # MySQL sample: 'DBI:mysql:database=translations;host=localhost;port=3306'
+        #
+        # Note 1: SQLite database will be created automatically
+        # if the target file does not exist
+        # Note 2: Database will be initialized automatically if necessary
+        db_source                   DBI:SQLite:dbname=../db/nycTest.db3
+
+        # (STRING) [OPTIONAL] Username for database connection
+        # (when applicable; has no effect for SQLite-based databases)
+        db_username                 l10n
+
+        # (STRING) [OPTIONAL] Password for database connection
+        # (when applicable;  has no effect for SQLite-based databases)
+        db_password                 secretword
+
+        # (STRING) Internal namespace to register files under in the database.
+        # Namespace (can be perceived as a localization project name) are used to group jobs
+        # together and are used to improve the fidelity of TM reuse: by default, translations
+        # are looked up in the same file, then in the same namespace, then in the entire TM
+        # database.
+        # Note that (namespace + relative file path) pair must be unique. So different jobs
```

Comment:
Whose responsibility will it be to make sure we don't have file path collisions?